website/docs/components/form/key-value-inputs/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Key Value Inputs
+description: >-
+  Key value inputs are rows of inputs that can be added and deleted by end users.
+caption: Key value inputs are rows of inputs that can be added and deleted by end users.
+links:
+  figma: >-
+    https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?node-id=67397-98270&t=w8xQlWxzH7bwXLe2-1
```

Comment:
I think this link is currently broken, you might need to update it when you actually merge the component branch.